### PR TITLE
Added dummy rule (2 version)

### PR DESCRIPTION
**Pull request by kewtree1408**

### CODE REVIEW

File: src/pslLint/elementsConventionChecker.ts [hunk 1]
```diff
@@ -25,6 +25,27 @@ export class PropertyStartsWithZ implements PropertyRule {
 	}
 }
 
+export class PropertyIsDummy implements PropertyRule {
+
+	ruleName = PropertyIsDummy.name;
+
+	report(_parsedDocument: PslDocument, property: Property): Diagnostic[] {
+		const diagnostics: Diagnostic[] = [];
+		if (_parsedDocument.parsedDocument.extending == null ) {
```
Comment (atiplea) [inline src/pslLint/elementsConventionChecker.ts]: Better:
```suggestion
		if (!parsedDocument.parsedDocument.extending) {
```

File: src/pslLint/elementsConventionChecker.ts [hunk 2]
```diff
@@ -25,6 +25,27 @@ export class PropertyStartsWithZ implements PropertyRule {
 	}
 }
 
+export class PropertyIsDummy implements PropertyRule {
+
+	ruleName = PropertyIsDummy.name;
+
+	report(_parsedDocument: PslDocument, property: Property): Diagnostic[] {
```
Comment (atiplea) [inline src/pslLint/elementsConventionChecker.ts]: The name `_parsedDocument` implies it is not being used, but here in this method it is. Please remove the `_`.

File: src/pslLint/elementsConventionChecker.ts [hunk 3]
```diff
@@ -25,6 +25,27 @@ export class PropertyStartsWithZ implements PropertyRule {
...
+		if (_parsedDocument.parsedDocument.extending == null ) {
+			this.isCalledDummy(property, diagnostics);
+		}
+		return diagnostics;
+	}
+
+	isCalledDummy(member: Member, diagnostics: Diagnostic[]): void {
+		if (member.id.value.toLowerCase() === 'dummy') {
+			diagnostics.push(
+				createDiagnostic(member, `property was called 'dummy'.`, DiagnosticSeverity.Information, this.ruleName)
```
Comment (atiplea) [inline src/pslLint/elementsConventionChecker.ts]: Better message: `Usage of "dummy" property is discouraged.`

File: __tests__/convention-test.ts [hunk 4]
```diff
@@ -23,12 +29,20 @@ describe('Members tests', () => {
...
 	})
 
 	test('Starts with v', () => {
 		expect(utils.diagnosticsOnLine(23, vDiagnostics).length).toBe(1);
-	})
-})
+	});
+
+	test('property was not called \'dummy\'', () => {
+		expect(utils.diagnosticsOnLine(2, withoutDummyDiagnostics).length).toBe(0);
```
Comment (atiplea) [inline __tests__/convention-test.ts]: No need to limit this test to a specific line. Check all the PropertyIsDummy diagnostics for the document to be 0.
```suggestion
		expect(withoutDummyDiagnostics.length).toBe(0);
```

**Comment by atiplea** [comment]
Great change Viktoriia. Please consider my comments. Thanks!

**Comment by kewtree1408** [comment]
Thanks! I fixed the code due to your review comments. Can you please check again? :-)

**Pull request merged**